.github/issue-labeler.yml

```diff
@@ -0,0 +1,3 @@
+# Add/remove 'support' label if issue contains a Jira Support issue id
+support:
+  - '(SUPPORT|SEC)-\d+'
```

Comment:
should trigger on just id's mentioned but also the links like `https://jira.camunda.com/browse/SUPPORT-99999`